contracts/contracts/vault/VaultCore.sol

```diff
@@ -109,7 +102,7 @@ contract VaultCore is VaultStorage {
         }
         // Rebase must happen before any transfers occur.
         if (priceAdjustedTotal > rebaseThreshold && !rebasePaused) {
```

Comment:
not fully fleshed out in my mind, but thinking - why is the rebaseThreshold at 1000e18 (seems low?) - what if I just keep doing flashloans and keep depositing and causing a rebase on redeem. Then the proportions change up on withdraw - and in this way i can bleed out the vault of more valuable token. Will think some more of pulling this attack off

Reply:
Both the mint and the redeem do a rebase if the amount is over $1,000. So I don't think flashloans aren't super helpful? Anyone can cause a rebase to happen anytime - it's not required to deposit or withdraw.

As of this morning a mint/redeem combo is around $25 USD in gas. Along with a withdrawal fee that works out to $5 , that means that in order to abuse selectively rebasing under, I think that means that there needs to be more than 3% of the value of the contract in unrebased yield in order to break even on minting in, rebasing, and redeeming out. That's a whole lot of yield. Even at 20% APY, that's about two months of going without a rebase.

Reply:
It's definitely worth thinking about the proportions.

Reply:
I beat on it for a while, didn't find a bug there. Still be glad if you can find something.